openai/cli.py

```diff
@@ -729,6 +724,17 @@ def help(args):
         help="A message in `{role} {content}` format. Use this argument multiple times to add multiple messages.",
         required=True,
     )
+
+    opt.add_argument(
+        "-e",
+        "--engine",
+        help="The engine to use. See https://platform.openai.com/docs/engines for more about what engines are available.",
+    )
+    opt.add_argument(
```

Comment:
Making both of them optional is fine too, but using a mutually exclusive group might be better: https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
Sure, updated.